src/import-data/ImportDataDestination.ts

```diff
@@ -219,6 +220,12 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
           ]),
       ]),
       importMayTakeTime && div({ style: { marginTop: '0.5rem', lineHeight: '1.5' } }, [importMayTakeTimeMessage]),
+      !!selectedWorkspace &&
+        h(Fragment, [
+          h(WorkspacePolicies, { workspace: selectedWorkspace }),
+          !!isProtectedData &&
```

Comment:
Do we intend to show this for GCP workspaces? I don't think policies work on GCP workspaces.

Reply:
Just to clarify the `this` you're referring to...do you mean the `WorkspacePolicies` component or the extra text about how we _may_ add additional access controls?

I asked @cahrens about this and while she is showing the `WorkspacePolicies` component for both GCP and Azure workspaces in the `ShareWorkspaceModal`, she only shows it for Azure in the `NewWorkspaceModal`.

She suggested I run this by @bethsheets for input.

Reply:
Thanks for the extra clarification.  So would it make sense to only show the notice indicating that it may add additional access controls when it's an Azure workspace, but otherwise leave the logic unchanged?

Reply:
I think so.

Reply:
I think it makes sense _not_ to display the 'we might mess with your workspace' messaging if that's not gonna happen for GCP workspaces.  So as to keep moving forward, I'll plan to only display it for Azure workspaces and push up a fix for that.

@bethsheets if you think this is incorrect, LMK and I'll either fix it here or in a followup if this lands first

Reply:
@nawatts I fixed this in 3a7d8de20a43aaa1e595adf265cd3bf5c64078e3, which also includes some test fixture cleanup which could probably use another look.

Reply:
@jladieu correct, this should only be for Azure workspaces. In GCP, the workspace is required to have additional security monitoring and we only show compliant workspaces. 

Here, for Azure, we are changing the permissions on the workspace when importing data by adding data access controls, so we are trying to tell them that. We also included the additional security monitoring because we thought it could be confusing for workspaces to flip between showing all or only some policies at different points in the experience.